### PR TITLE
Added a new sample to show map load status

### PR DESCRIPTION
**Pull request by manishk3189**
The sample demonstrates how to add a load status change listener on Loadable map and get the map load status. [#985](https://devtopia.esri.com/runtime/java-android-sdk/issues/985)

@doneill  Can you please review the PR and let me know of changes if required?

Thanks

### CODE REVIEW

File: map-loaded/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <color name="primary">#2196F3</color>
+    <color name="primary_dark">#1976D2</color>
+    <color name="accent">#FF4081</color>
+    <color name="text_primary">#212121</color>
```
Comment (doneill) [inline map-loaded/src/main/res/values/colors.xml]: appears to be unused, remove.

File: map-loaded/src/main/res/values/colors.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+    <color name="text_secondary">#727272</color>
```
Comment (doneill) [inline map-loaded/src/main/res/values/colors.xml]: appears to be unused, remove

File: map-loaded/src/main/java/com/esri/arcgisruntime/sample/maploaded/MainActivity.java [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+import com.esri.arcgisruntime.loadable.LoadStatusChangedListener;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.view.MapView;
+
+public class MainActivity extends AppCompatActivity {
+
+    private static String TAG = "MapLoadStatus";
+    private MapView mMapView;
+    private ArcGISMap mMap;
```
Comment (doneill) [inline map-loaded/src/main/java/com/esri/arcgisruntime/sample/maploaded/MainActivity.java]: Convert to local field.

File: map-loaded/src/main/java/com/esri/arcgisruntime/sample/maploaded/MainActivity.java [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+
+import com.esri.arcgisruntime.loadable.LoadStatusChangedEvent;
+import com.esri.arcgisruntime.loadable.LoadStatusChangedListener;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.view.MapView;
+
+public class MainActivity extends AppCompatActivity {
+
+    private static String TAG = "MapLoadStatus";
```
Comment (doneill) [inline map-loaded/src/main/java/com/esri/arcgisruntime/sample/maploaded/MainActivity.java]: Make declaration Final.

**Comment by doneill** [comment]
Remove the `androidTest` and `Test` sourceSets as they are not implemented.

**Comment by doneill** [comment]
@mani8177 I have some early comments, more may come as I review further, please address in the mean time.

**Comment by manishk3189** [comment]
@doneill I have accommodated the first set of changes as per your review comments. Will update other PRs as well for same review comments

File: map-loaded/build.gradle [hunk 5]
```diff
@@ -20,7 +20,5 @@ android {
 }
 
 dependencies {
-    compile fileTree(dir: 'libs', include: ['*.jar'])
-    testCompile 'junit:junit:4.12'
     compile 'com.android.support:appcompat-v7:23.4.0'
```
Comment (doneill) [inline map-loaded/build.gradle]: Update this to the latest version v24.0.0

File: map-loaded/src/main/java/com/esri/arcgisruntime/sample/maploaded/MainActivity.java [hunk 6]
```diff
@@ -32,9 +32,9 @@
 
 public class MainActivity extends AppCompatActivity {
 
-    private static String TAG = "MapLoadStatus";
+    private static final String TAG = "MapLoadStatus";
+    ArcGISMap mMap;
```
Comment (doneill) [inline map-loaded/src/main/java/com/esri/arcgisruntime/sample/maploaded/MainActivity.java]: Make this field private

File: map-loaded/README.md [hunk 7]
```diff
@@ -0,0 +1,59 @@
...
+
+![screen shot 2016-05-27 at 3 01 36 pm](https://cloud.githubusercontent.com/assets/12448081/15622274/fa2fe8f2-241b-11e6-9cdc-2db8ca96dbf8.png)
+#Features
+
+* Map
+* MapView
+* LoadStatusChangedListener
+
+# Developer Pattern
+
```
Comment (doneill) [inline map-loaded/README.md]: Put a few comments as to what the pattern is.

**Comment by doneill** [comment]
@manishk3189 Looking good, please address the remaining minor comments and I will look to merge.

**Comment by manishk3189** [comment]
Hi @doneill I have accommodated the review comments.

**Pull request merged**